static/js/apps/nl_interface/app.tsx

```diff
@@ -184,7 +177,6 @@ export function App(): JSX.Element {
             }
           }}
         />
-        {isStartState && <QueryHistory onItemClick={onHistoryItemClick} />}
```

Comment:
wondering if this is something we're just temporarily getting rid of & eventually want back? 

because if we will want to add this back, we could just replace `isStartState` with `false` instead of deleting everything?

Reply:
yes, it seems temporary - sending you a follow up (sorry for premature auto-merge)